CMakeLists.txt

```diff
@@ -101,10 +107,12 @@ target_link_libraries(sarbp_test PRIVATE Halide::Halide
                                          ip_u_hat
                                          ip_pixel_locs
                                          backprojection
+                                         backprojection_distributed
                                          backprojection_cuda
                                          backprojection_ritsar
                                          backprojection_ritsar_s
                                          backprojection_ritsar_p
                                          backprojection_ritsar_vp
                                          img_output_u8
-                                         img_output_to_dB)
+                                         img_output_to_dB
+                                         ${MPI_CXX_LIBRARIES})
```

Comment:
Pedantic: please move this up with other external libs, e.g., after `${FFTW_STATIC_LIBRARIES}`

Reply:
Required: Add `${MPI_CXX_INCLUDE_PATH}` to `target_include_directories` for `sarbp_test`